src/QuokkaSimulation.hpp

```diff
@@ -2404,6 +2429,7 @@ void QuokkaSimulation<problem_t>::advanceRadiationSubstepAtLevel(int lev, amrex:
 	}
 
 	if (do_reflux) {
+		amrex::Gpu::streamSynchronizeAll();
```

Comment:
This is actually unnecessary. I'll remove this and the subsequent ones in this file.